src/twisted/application/runner/test/test_runner.py

```diff
@@ -68,11 +68,31 @@ def test_runInOrder(self):
         """
         L{Runner.run} calls the expected methods in order.
         """
-        runner = DummyRunner({})
+
+        calledMethods = []
+
+        self.patch(
```

Comment:
I'm having a hard time caring either way, but I don't want to switch this back because of the whim-du-jour of the reviewer, even if it's @glyph.

What's the problem with patching here that you want to avoid?  There's a lot of patching here that I can't simply replace with subclasses, so I don't get why this case is special, or are you asking me to rewrite everything that calls `self.patch`?